First_project/First_project/Profile/LoginViewController.swift

```diff
@@ -0,0 +1,218 @@
+            loginButton.trailingAnchor.constraint(equalTo: areaView.trailingAnchor, constant: AppConstants.rightAnchorSize),
+            loginButton.heightAnchor.constraint(equalToConstant: 50),
+            loginButton.bottomAnchor.constraint(equalTo: scrollView.bottomAnchor)
+        ])
+        
+        profileLogin.layer.maskedCorners = [.layerMinXMinYCorner, .layerMaxXMinYCorner]
+        profilePassword.layer.maskedCorners = [.layerMaxXMaxYCorner, .layerMinXMaxYCorner]
+        
+        navigationController?.setNavigationBarHidden(true, animated: true)
+        
+    }
+    
+    // MARK Keyboard observers
+    override func viewDidDisappear(_ animated: Bool) {
+        super.viewDidDisappear(animated)
+        
+        NotificationCenter.default.removeObserver(self, name: UIResponder.keyboardWillShowNotification, object: nil)
+        NotificationCenter.default.removeObserver(self, name: UIResponder.keyboardWillHideNotification, object: nil)
+    }
+    
+    // MARK keyboard actions
+    @objc fileprivate func keyboardWillShow(notification: NSNotification) {
+        
+        if let keyboardSize = (notification.userInfo?[UIResponder.keyboardFrameEndUserInfoKey] as? NSValue)?.cgRectValue {
+            
+            scrollView.contentInset.bottom = keyboardSize.height
+            scrollView.verticalScrollIndicatorInsets = UIEdgeInsets(top: 0, left: 0, bottom: keyboardSize.height, right: 0)
+        }
+    }
+    
+    @objc fileprivate func keyboardWillHide(notification: NSNotification) {
+        
+        scrollView.contentInset.bottom = .zero
+        scrollView.verticalScrollIndicatorInsets = .zero
+    }
+    
+    @objc private func buttonPressed() {
+        let profileC = ProfileViewController()
+        self.navigationController?.pushViewController(profileC, animated: true)
+    }
+    
+    @objc private func buttonChanged(_ select: UIButton) {
+        select.alpha = 0.5
+        DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
+            select.alpha = 1.0
+        }
+    }
+    
+    @IBAction func iconAction(sender: AnyObject) {
+        if(iconicClick == true) {
+            profilePassword.isSecureTextEntry = false
+        } else {
+            profilePassword.isSecureTextEntry = true
+        }
+        iconicClick = !iconicClick
+    }
+    
+    @objc func keybordWillChange(notification: Notification) {
+        print("0")
+    }
+}
+
+extension LoginViewController: UITextFieldDelegate {
+    func textFieldShouldReturn(_ textField: UITextField) -> Bool {
+        if textField == self.profileLogin {
+            self.profilePassword.becomeFirstResponder()
+        }
+        if textField == self.profilePassword {
+            textField.resignFirstResponder()
+        }
+        return true
+    }
+}
+
+extension LoginViewController: UIScrollViewDelegate {
+    override func viewWillAppear(_ animated: Bool) {
```

Comment:
viewWillAppear and other UIViewController methods should be in the class, not in extension.
Usual class order:
properties,
computed properties,
lifecycle methods,
other methods
UI related methods.